Jackson/BaseScreen.html

```diff
@@ -0,0 +1,79 @@
+<!-- 
+- No business rules required
+- No css required
+- Create unit test to check if the elements are visible and has the text (maybe you have to add the id for the element)
+--> 
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>WOB BaseScreen</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+<style> 
+  * {
+    box-sizing: border-box;
+  }
+
+  .container {
+    border-radius: 5px;
+    border: 1 px solid black;
+    /*background-color: #f2f2f2;*/
+    background: url("supermarket.jpg");
```

Comment:
Also don't add the background now, keep it to be added in another task.
Therefore remove Jackson/supermarket.jpg